### PR TITLE
docs(README): explain what the loader does

### PR DESCRIPTION
**Pull request by simon04**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,7 +10,7 @@
       src="https://webpack.js.org/assets/icon-square-big.svg">
   </a>
   <h1>Mocha Loader</h1>
-  <h3>Mocha for webpack</h3>
+  <p>Allows <a href="http://mochajs.org/">Mocha</a> tests to be loaded via webpack</p>
```
Comment (simon04) [inline README.md]: loaded? run?

Comment (michael-ciniawsky) [inline README.md]: mom :)

Comment (michael-ciniawsky) [inline README.md]: both 😛

Comment (simon04) [inline README.md]: Thanks for clarifying. I amended the commit.

**Pull request merged**